app/hub/Views/SetupView/SetupViewContainer.jsx

```diff
@@ -104,7 +104,7 @@ class SetupViewContainer extends Component {
 		this.props.actions.setAntiTracking({ enable_anti_tracking: true });
 		this.props.actions.setAdBlock({ enable_ad_block: true });
 		this.props.actions.setSmartBlocking({ enable_smart_block: true });
-		this.props.actions.setGhosteryRewards({ enable_ghostery_rewards: true });
+		this.props.actions.setGhosteryRewards({ enable_ghostery_rewards: !IS_FIREFOX });
```

Comment:
This isn't your code at all, so I will approve this PR, but I want to take this opportunity to express my sadness about calling this action via `_setDefaultSettings()` in the `constructor()` of this component.

Reply:
I want to take this opportunity to express that I share your sadness.